docs/configuration.md

```diff
@@ -42,8 +42,9 @@ ecosystem = "Go"
 group = "dev"
 
 # Actions to take for matching packages:
-ignore = true # Ignore this package entirely, including license scanning
-license.override = ["MIT", "0BSD"] # Override the license of the package, if it is not ignored
+ignore = true # Ignore this package completely, including license scanning
+license.ignore = true # Ignore the license of the package, if it is not already completely ignored at the top level
+license.override = ["MIT", "0BSD"] # Override the license of the package, if it is not completely ignored at the top level
```

Comment:
note: I've purposely kept the documentation here light as I think it'll be easier to document once #1226 is landed